samples/powerpoint/custom-functions/search-wikipedia-api.yaml

```diff
@@ -0,0 +1,221 @@
+order: 1
+id: powerpoint-custom-functions-search-wikipedia-api
+name: Search Wikipedia
+description: Search Wikipedia based on the selected text in the presentation.
+author: cakriwut
+host: POWERPOINT
+api_set: {}
+script:
+    content: |
+        declare var moment: any;
+        $("#search").click(run);
+
+        async function run() {
+          Office.context.document.getSelectedDataAsync(Office.CoercionType.Text, {}, getSelectedText);
+        }
+
+        function getSelectedText(result) {
```

Comment:
In order to help others better learn from your snippet, could you add a brief description before each function? A short sentence or two will help people understand what your code is doing and parse it faster.

Reply:
Updated.